src/lib/UniversalDiscovery/Event/ConfigResolveEvent.php

```diff
@@ -14,6 +14,8 @@ class ConfigResolveEvent extends Event
 {
     public const NAME = 'udw.resolve.config';
 
+    public const READ_SPECIFIC_CONFIGURATIONS = ['richtext_embed', 'richtext_embed_image', 'browse'];
```

Comment:
I see this only used in context of checking whether the config name matches or does not match entries in this array.

If that's the case, I think it's more appropriate to introduce an event method instead?